gateway-ha/src/main/java/io/trino/gateway/ha/config/DataStoreConfiguration.java

```diff
@@ -20,14 +20,16 @@ public class DataStoreConfiguration
     private String password;
     private String driver;
     private Integer queryHistoryHoursRetention = 4;
+    private Integer cookieHoursRetention = 24;
 
-    public DataStoreConfiguration(String jdbcUrl, String user, String password, String driver, Integer queryHistoryHoursRetention)
+    public DataStoreConfiguration(String jdbcUrl, String user, String password, String driver, Integer queryHistoryHoursRetention, Integer cookieHoursRetention)
     {
         this.jdbcUrl = jdbcUrl;
         this.user = user;
         this.password = password;
         this.driver = driver;
         this.queryHistoryHoursRetention = queryHistoryHoursRetention;
+        this.cookieHoursRetention = cookieHoursRetention;
```

Comment:
cookieRetentionTime

or 

cookieDuration

I think this should use the airlift "duration" as unit ideally .. not "hours"